src/common/transformations/tests/common_optimizations/convert_compressed_to_mixed_precision_test.cpp

```diff
@@ -0,0 +1,177 @@
+    {
+        auto input = std::make_shared<ov::opset8::Parameter>(ov::element::f32, ov::Shape{1, 3, 12, 12});
+        auto const_weights = ov::opset8::Constant::create(
+            ov::element::f32,
+            ov::Shape{1, 3, 3, 3},
+            {1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9});
+        auto conv = std::make_shared<ov::opset8::Convolution>(input,
+                                                              const_weights,
+                                                              ov::Strides{1, 1},
+                                                              ov::CoordinateDiff{0, 0},
+                                                              ov::CoordinateDiff{0, 0},
+                                                              ov::Strides{1, 1});
+
+        f = std::make_shared<ov::Model>(ov::NodeVector{conv}, ov::ParameterVector{input});
+
+        ov::pass::Manager manager;
+        manager.register_pass<ov::pass::InitNodeInfo>();
+        manager.register_pass<ov::pass::ConvertCompressedToMixedPrecision>();
+        manager.run_passes(f);
+        ASSERT_NO_THROW(check_rt_info(f));
+    }
+
+    {
+        auto input = std::make_shared<ov::opset8::Parameter>(ov::element::f32, ov::Shape{1, 3, 12, 12});
+        auto const_weights = ov::opset8::Constant::create(
+            ov::element::f32,
+            ov::Shape{1, 3, 3, 3},
+            {1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9});
+        auto conv = std::make_shared<ov::opset8::Convolution>(input,
+                                                              const_weights,
+                                                              ov::Strides{1, 1},
+                                                              ov::CoordinateDiff{0, 0},
+                                                              ov::CoordinateDiff{0, 0},
+                                                              ov::Strides{1, 1});
+
+        f_ref = std::make_shared<ov::Model>(ov::NodeVector{conv}, ov::ParameterVector{input});
+    }
+
+    auto res = compare_functions(f, f_ref, true);
+    ASSERT_TRUE(res.first) << res.second;
+}
+
+template <element::Type_t T>
+bool has_type(std::shared_ptr<Model> model) {
+    for (auto& node : model->get_ordered_ops()) {
+        for (auto& input : node->inputs()) {
+            if (input.get_element_type() == element::Type(T)) {
+                return true;
+            }
+        }
+        for (auto& output : node->outputs()) {
+            if (output.get_element_type() == element::Type(T)) {
+                return true;
+            }
+        }
+    }
+    return false;
+}
+
+TEST(TransformationTests, ConvertCompressedToMixedPrecission_keep_sensitive_nodes_in_fp32) {
+    std::shared_ptr<ov::Model> model(nullptr), model_ref(nullptr);
+    {
+        auto input_1 = std::make_shared<opset8::Parameter>(element::f32, Shape{360, 640});
```

Comment:
please use `opset10` everywhere in the tests

Reply:
agreed and renamed